src/app/components/core/control-dispatcher/pipelines/control-pipelines.component.html

```diff
@@ -0,0 +1,95 @@
+<div class="container is-fluid">
+  <app-add-dispatcher-service></app-add-dispatcher-service>
+  <div class="card">
+    <header class="card-header">
+      <div class="card-header-title">
+        Control Pipelines
+        <button class="button is-small" id="refresh-check" (click)="getControlPipelines()">
+          <i class="fa fa-sm fa-sync" aria-hidden="true"></i>
+        </button>
+      </div>
+      <a *ngIf="rolesService?.hasEditPermissions()" class="fix-pad button is-light" (click)="addControlPipeline()">
+        <p *ngIf="viewPort !== 'mobile'" class="add-btn">Add &nbsp;</p>
+        <i class="fa fa-xs fa-plus" aria-hidden="true"></i>
+      </a>
+    </header>
+    <div class="card-content">
+      <div class="content">
+        <div *ngIf="showSpinner" class="app-loading">
+          &nbsp;
+          <svg class="spinner" viewBox="0 0 100 100">
+            <circle class="path" cx="50%" cy="50%" r="10" fill="none" stroke-width="2" stroke-miterlimit="10" />
+          </svg>
+        </div>
+        <ng-container *ngIf="!showSpinner">
+          <div class="has-text-centered" *ngIf="pipelines != undefined && pipelines.length === 0">
+            <small class="no-rec">No Record</small>
```

Comment:
No Control Pipelines found

And should be in centre of container; Same for ACLs and Scripts